CONTRIBUTING.md

```diff
@@ -29,16 +31,41 @@ Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe
 disclosure of security bugs. In those cases, please go through the process
 outlined on that page and do not file a public issue.
 
+## Environment setup
+
+```
+~$ python3 -m venv venv2
+~$ source venv2/bin/activate
+(venv2) ~$ cd git/fairscale/
+(venv2) ~/git/fairscale (min/setup)$ pip3 install -r requirements-test.txt
```

Comment:
maybe remove the bash prompt from here, especially the min/setup part. It might be confusing.

Reply:
good catch. will do.